app/assets/javascripts/transactions.js

```diff
@@ -51,26 +46,41 @@ $(function () {
     $('#new_transaction').show();
   });
 
+  $(document).on('change', '#transfer_amount', function(e) {
+    prep_rate_and_hints(exchange_rate = false, hints = true)
```

Comment:
@Gataniel please clarify what this means? `prep_rate_and_hints(exchange_rate = false, hints = true)` - I think it's incorrect passing values to function.